test/inductor/test_torchinductor.py

```diff
@@ -5017,6 +5017,11 @@ def fn(x):
             ],
         )
 
+    @unittest.skip(
+        """
+        FIX: this test is flaky
```

Comment:
file a bug for it and link?

Reply:
I tried to repro again locally in order to include an error in the issue, and realized I can't repro anymore :(. I think I'll remove the skip and see if CI passes for sanity (and if it does, I'll just land this PR - since the flakiness of this test is independent). If it comes up again, I'll skip and file an issue.